projects/ad9361/src/parameters.h

```diff
@@ -148,6 +148,9 @@
 #define CF_AD9361_TX_DMA_BASEADDR	3
 
 #define SPI_DEVICE_ID		0
+#define DAC_DDR_BASEADDR	0x10000000
```

Comment:
This may be an issue. In linux you can't access raw memory address. 
Try using an approach like this: https://github.com/analogdevicesinc/no-OS/blob/b34afe2704b607c0789b91b25a0f875236616a0b/projects/iio_demo/src/app/main.c#L86